pkg/analytics/instrumentation_collector.go

```diff
@@ -121,22 +124,72 @@ func (ic *instrumentationCollectorImpl) AddExtension(key string, value interface
 	ic.extension[key] = value
 }
 
-func GetV2InstrumentationObject(collector InstrumentationCollector) (*api.AnalyticsRequestBody, error) {
+func GetV2InstrumentationObject(logger *zerolog.Logger, collector InstrumentationCollector) (*api.AnalyticsRequestBody, error) {
```

Comment:
Question: I think this interface change might break the current integration in snyk-ls. You might see this when building the CLI with this gaf version.

Reply:
Got it. Maybe I need another approach.